packages/macros/src/index.ts

```diff
@@ -35,12 +35,12 @@ export function importSync(specifier: string): unknown {
   throw new Oops(specifier);
 }
 
-export function getConfig<T>(packageName: string): T {
-  throw new Oops(packageName);
+export function getConfig<T>(packageName: string, ...keys: string[]): T {
```

Comment:
These are the types for the Javascript macros, not the template macros. The Javascript `getConfig` doesn't take any `keys` argument (because it works with regular property access and optional chaining instead).

So probably this change should go in `class macroGetConfig extends Helper` instead, which would change to declare its own type and not rely on this type, since they're supposed to be different.

Reply:
removed these parameters and use them only in template types